packages/database/src/database/events.ts

```
@@ -0,0 +1,60 @@
+import "server-only"
+import { prismaClient } from ".."
+import { Prisma, Events } from "@prisma/client"
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)